src/main/java/frc/robot/commands/States/Eject.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.States;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.constIntake;
+import frc.robot.Constants.constTransfer;
+import frc.robot.RobotContainer.RobotState;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.StateMachine;
+import frc.robot.subsystems.Transfer;
+
+public class Eject extends Command {
+  StateMachine subStateMachine;
+  Intake subIntake;
+  Transfer subTransfer;
+
+  /** Creates a new Eject. */
+  public Eject(Intake subIntake, Transfer subTransfer, StateMachine subStateMachine) {
```

Comment:
We should probably try to keep the order of subsystem parameters consistent. I recommend subStateMachine first, then alphabetical for the rest. Which looks like what you've already done except for this file:
<img width="753" alt="Screenshot 2024-08-12 at 9 38 56 AM" src="https://github.com/user-attachments/assets/58b60cb0-e1c5-4465-9d44-0fae839e663e">